server/src/main/java/io/spine/server/bus/Bus.java

```diff
@@ -71,18 +81,55 @@ public void unregister(D dispatcher) {
     /**
      * Posts the message to the bus.
      *
-     * @param message the message to post
+     * <p>Use the {@code Bus} class abstract methods to modify the behavior of posting.
+     *
+     * @param message          the message to post
+     * @param responseObserver the observer to receive outcome of the operation
+     */
+    // Left non-final for testing purposes.
```

Comment:
As a second thought, let's append the method Javadoc with a notion, that this method defines a general flow for a message in a bus and should not be considered as overridable. And remove the comment.

Also, I would create an issue (tech debt) saying that this method should be made `final`, once we get rid of Mockito or if Mockito behavior changes.